container/Dockerfile

```diff
@@ -49,7 +49,7 @@ RUN ${IOT_REPO_PATH}/rest/install.sh
 
 # Copy ES2.0 client libraries
 RUN echo "Downloading Event Store 2.0 client libraries, it will take some time..."
-RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.2.1/2.0.0/ibm-db2-eventstore-client-spark-2.2.1-2.0.0.jar
+RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar
```

Comment:
Can you change the comment and echo to say `2.0.1.0` instead of `2.0`

Reply:
done